packages/office-ui-fabric-react/src/demo/pages/DetailsListPage/examples/DetailsList.Advanced.Example.tsx

```diff
@@ -441,6 +442,11 @@ export class DetailsListAdvancedExample extends React.Component<any, IDetailsLis
   }
 
   @autobind
+  private _onItemContextMenu(item: any, index: number) {
+    console.log('Item context menu invoked', item, index);
```

Comment:
Not quite sure if you missed a commit since this method doesn't call any callbacks.